setup_env.py

```diff
@@ -2,10 +2,12 @@
 import logging
 import os
 
-from diem import testnet, chain_ids
+from diem import testnet, LocalAccount, jsonrpc
 
 liquidity_account_name = "liquidity"
 
+PREMAINNET_CHAIN_ID = 21
```

Comment:
Why are we hardcoding this? We should be reading it from ENV?

Reply:
Because I needed a way to understand if I am configuring for premainnet. Even if I assume I get it from the environment, I'll still have to write something like `if chain_id == PREMAINNET_CHAIN_ID` and since the ID is not defined in the SDK anymore, there are no good options. I'd appreciate any suggestion on how it could be improved.

Reply:
Not ideal because we will need to update this whenever there is new premainet deployment and also update the deployment script too. I would prefer to keep the chain ID information strictly to helm configs. 

"Because I needed a way to understand if I am configuring for premainnet." <- doesn't this get solved if you pass in BLOCKCHAIN? you know you're configuring for premainnet if blockchain == premainnet

Reply:
I am not sure I understand. There are two variables at play here: 

1. On which blockchain I want to run.
2. What it the chain ID of premainnet. 
 
Do you prefer both to be configured externally? In the meantime, I'll change PREMAINNET_CHAIN_ID to be read from an environment variable and use 21 as a fallback only. Will push in a minute.

Reply:
Yeah think it makes more sense that chain ID is configured externally from env var. I actually think we should remove the fallback as well since there should only be a single source of truth

Reply:
This affects development environment only, not a value needed in the helm charts, for example. And there is no other source of truth in dev env.

Reply:
I guess if we know exactly what the chain ID needs to be why do we need a default fallback value? I might be missing something here

Reply:
My reasoning is that this script will be executed by some developers on their local machines and I prefer to save them the hassle of knowing that they have to define something in their environment. I assume premainnet ID will be constant eventually, so it won't be an issue. If the ID changes again and we forget to update it here, it won't make the situation worse for those developers, but if we do update, it will be much more convenient for them.